examples/audio/audio_music_stream.c

```diff
@@ -23,6 +26,7 @@ int main(void)
     InitAudioDevice();              // Initialize audio device
 
     Music music = LoadMusicStream("resources/country.mp3");
+    music.background = AUDIO_THREAD_MUSIC_UPDATE;
```

Comment:
This mechanism looks very weird from an user perspective, I'd prefer to avoid it and look for an alternative.